pipelines/tekton/validate.go

```diff
@@ -23,16 +24,23 @@ func (e ErrRuntimeNotSupported) Error() string {
 }
 
 func validatePipeline(f fn.Function) error {
-	if f.Runtime == "" {
-		return ErrRuntimeRequired
-	}
-
-	if f.Runtime == "go" || f.Runtime == "rust" {
-		return ErrRuntimeNotSupported{f.Runtime}
-	}
-
-	if len(f.Buildpacks) > 0 {
-		return ErrBuilpacksNotSupported
+	if f.Builder == fn.BuilderPack {
+		if f.Runtime == "" {
+			return ErrRuntimeRequired
+		}
+
+		if f.Runtime == "go" || f.Runtime == "rust" {
```

Comment:
What about a check here for Python and SpringBoot, and test cases below?

Reply:
I think that both are functional, @jrangelramos is that right?

Reply:
Yes @zroubalik Python and Springboot are functional